nengo_gui/static/menu.js

```diff
@@ -27,6 +27,8 @@ Nengo.Menu.prototype.show = function (x, y, items) {
         return;
     }
 
+    this.menu_margin = 15;
+
```

Comment:
> What happens in firefox if the window is fullscreen so it can't overflow out the bottom? (indeed, all of these checks should be in situations where the window is fullscreen so there's no option for the right-click menu to overflow)

This was in the case of it being full screen, it would simply spill over onto the taskbar.